package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "autobahn",
-  "version": "19.7.2",
+  "version": "19.7.3",
   "description": "An implementation of The Web Application Messaging Protocol (WAMP).",
   "main": "index.js",
+  "private": true,
```

Comment:
This `package.json` is only for local development and CI usage. `"private": true"` makes it safe from accidental `npm publish`.

Reply:
Neat .. wasn't aware of that one ..